CHANGELOG.md

```diff
@@ -53,14 +53,41 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Removed unnecessary requests in `Management/Status` section. [#5528](https://github.com/wazuh/wazuh-kibana-app/pull/5528)
 - Removed obsolete code that caused duplicate requests to the api in `Management`. [#5485](https://github.com/wazuh/wazuh-kibana-app/pull/5485)
 
+## Wazuh v4.5.1 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 01
+
+### Added
+
+- Add Apple Silicon architecture button to the register Agent wizard [#5478](https://github.com/wazuh/wazuh-kibana-app/pull/5478)
+
+### Fixed
+
+- Fixed the rendering of tables that contains IPs and agent overview [#5471](https://github.com/wazuh/wazuh-kibana-app/pull/5471)
+- Fixed the agents active coverage stat as NaN in Details panel of Agents section [#5490](https://github.com/wazuh/wazuh-kibana-app/pull/5490)
+
+### Removed
+
+- Removed the agent name in the agent info ribbon [#5497](https://github.com/wazuh/wazuh-kibana-app/pull/5497)
+
+### Changed
+
+- Changed method to perform redirection on agent table buttons [#5539](https://github.com/wazuh/wazuh-kibana-app/pull/5539)
+- Changed windows agent service name in the deploy agent wizard [#5538](https://github.com/wazuh/wazuh-kibana-app/pull/5538)
+
 ## Wazuh v4.5.0 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 01
 
 ### Added
 
 - Support for Wazuh 4.5.0
 
+## Wazuh v4.4.5 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 02
+
+### Added
+
+- Support for Wazuh 4.4.5
+
 ## Wazuh v4.4.4 - Kibana 7.10.2, 7.16.x, 7.17.x - Revision 01
 
+
```

Comment:
**nitpick**: remove the new line.

Reply:
Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f